README.md

```diff
@@ -17,41 +17,91 @@ Please visit the [latest stable release](https://github.com/Open-CMSIS-Pack/cpac
+
+```
+$ cpackget init --pack-root path/to/new/pack-root https://keil.com/pack/index.pidx
+```
+
+The command will create a folder called `path/to/new/pack-root`, a few subfolders (`.Download`, `.Local`, `.Web`)
+and will place a copy of the index file (if specified) to `.Web/index.pidx`.
+
+If later it is needed to update the public index file, just run `cpackget index https://vendor.com/index.pidx` and
+`.Web/index.pidx` will be updated accordingly.
+
+
+### Adding packs
 
 The commands below are examples on how to add packs:
-- `cpackget pack add path/to/Vendor.PackName.x.y.z.pack`
-- `cpackget pack add https://vendor.com/example/Vendor.PackName.x.y.z.pack`
 
-If the pack is specified by an URL, cpackget command will download `Vendor.PackName.x.y.z.pack` before installing it under the **Open-CMSIS-Pack** root folder.
-This folder should be specified via system environment variable named **CMSIS_PACK_ROOT**. If empty, cpackget will attempt creating a folder called `.cpackget` in the local directory and that will become the installation folder.
+* `cpackget pack add path/to/Vendor.PackName.x.y.z.pack`
+* `cpackget pack add https://vendor.com/example/Vendor.PackName.x.y.z.pack`
+* `cpackget pack add Vendor.PackName.x.y.z`
+* `cpackget pack add Vendor.PackName`
+* `cpackget pack add -f list-of-packs.txt`, the conten is simply a list of packs, one per line
 
 The command below is an example how to add packs via PDSC files:
-- `cpackget pdsc add path/to/Vendor.PackName.pdsc`
+* `cpackget pdsc add path/to/Vendor.PackName.pdsc`
 
 Note that for adding packs via PDSC files is not possible to provide an URL as input. Only local files are allowed.
 
-Removing packs uses slightly different pack specification. Here are a few examples
-- `cpackget pack rm Vendor.PackName.x.y.z`: removes only the *x.y.z* version of the `Vendor.PackName` pack.
-- `cpackget pack rm Vendor.PackName`: removes all installed versions of the `Vendor.PackName` pack.
-- `cpackget pack rm --purge Vendor.PackName`: using `--purge` triggers removal of any downloaded files.
+### Bypassing End User License Agreement (EULA)
+
+Some packs come with licenses and by default cpackget will prompt the user for agreement. This can be avoided
+by using the `--agree-embedded-license` flag:
+
+* `cpackget pack add --agree-embedded-license Vendor.PackName`
+
+Also there are cases where users might want to only extract the pack's license and not install it:
+
+* `cpackget pack add --extract-embedded-license Vendor.PackName`
+
+The extracted license file will be placed next to the pack's. For example if Vendor.PackName.x.y.z had a licese file
+named `LICENSE.txt`, cpackget would extract it to `.Download/Vendor.PackName.x.y.z.LICENSE.txt`.
+
+### Removing packs
+
+Removing packs uses slightly different pack specification. Here are a few examples:
+
+* `cpackget pack rm Vendor.PackName.x.y.z`: removes only the *x.y.z* version of the `Vendor.PackName` pack.
```

Comment:
Probably should align the style. First a text line explaining what the command will do and then the command line example.

Reply:
ack